File: src/main/java/com/tatsuaki/carepreventioncsv/model/domain/CarePreventionCsvColumn/SpareColumn.java
package com.tatsuaki.carepreventioncsv.model.domain.CarePreventionCsvColumn;

public class SpareColumn {

    private final String spareColumn;

    public SpareColumn(String spareColumn) {
        this.spareColumn = spareColumn;
    }

    public String getSpareColumn() {
        return spareColumn;
    }

    public String getFormatErrorMessage() {
        // フリーテキストのためチェックなし
        return "";
    }
}

File: src/main/kotlin/com/tatsuaki/carepreventioncsv/model/domain/csvcolumn/SpareColumn.kt
package com.tatsuaki.carepreventioncsv.model.domain.csvcolumn

class SpareColumn(spareColumn: String) : CsvColumn(spareColumn) {

    override fun getFormatErrorMessage(): String {
        // フリーテキストのためチェックなし
        return ""
    }

    override fun getFormatContent(): String {
        return content
    }

    override fun validateCharacter(): Boolean {
        return true
    }

    override fun validateLength(): Boolean {
        return true
    }

    override fun validateCodeFormat(): Boolean {
        return true
    }
}
